benchmarks/benchmarks/ci/esmf_regridder.py

```diff
@@ -192,21 +186,22 @@ def setup(self, type):
             n_lons_tgt,
             n_lats_tgt,
             h,
-            coord_system_src,
-        ) = self.get_args(type)
-        mesh = _gridlike_mesh(n_lons_src, n_lats_src)
+            alt_coord_system_src,
+        ) = self.get_args(tp)
+        src_mesh = _gridlike_mesh_cube(n_lons_src, n_lats_src).mesh
```

Comment:
I'm a bit more reluctant to do this via `_gridlike_mesh_cube` if this would involve the caching of each of these cubes since they are likely to significantly increase the size of the files cached. I feel like this would involve even more saving/loading of large cubes which may be better to keep to a minimum.

Reply:
This makes sense. If anyone is wondering, the caching that @stephenworsley refers to is the fact that `_gridlike_mesh_cube()` goes via the scripts in `generate_data.py`, thus forcing us to use IO - an ugly quirk of any commit-regression benchmarking solution that needs synthetic data - so in the absence of a super-amazing benchmarking machine, the solution here is a good compromise.